### PR TITLE
Set mirror for kotlin

### PR DESCRIPTION
**Pull request by blindpirate**
### Context

This PR fixes https://github.com/gradle/gradle-private/issues/1414 and set mirrors for:

- https://dl.bintray.com/kotlin/kotlin-eap/
- https://dl.bintray.com/kotlin/kotlin-dev/

### CODE REVIEW

File: subprojects/resources-http/src/main/java/org/gradle/internal/resource/transport/http/HttpClientHelper.java [hunk 1]
```diff
@@ -121,6 +121,9 @@ public boolean wasSuccessful(CloseableHttpResponse response) {
     }
 
     public CloseableHttpResponse performHttpRequest(HttpRequestBase request) throws IOException {
+        if(request.getURI().toString().contains("dl.bintray.com/kotlin")) {
```
Comment (wolfs) [inline subprojects/resources-http/src/main/java/org/gradle/internal/resource/transport/http/HttpClientHelper.java]: I guess we shouldn't commit this change to `master`.

Comment (blindpirate) [reply subprojects/resources-http/src/main/java/org/gradle/internal/resource/transport/http/HttpClientHelper.java]: Yes, once this is approved, I'll remove this.

**Comment by wolfs** [comment]
@blindpirate Why did this change need to be reverted?

**Comment by blindpirate** [comment]
You mean this commit https://github.com/gradle/gradle/pull/6123/commits/8f92c4c074758d8ecfe0799eedc7b06c11ba2408 ? That's because `startParameter.projectProperties` is read-only and I can't add anything to it.

**Pull request merged**